tesseract_motion_planners/include/tesseract_motion_planners/descartes/profile/descartes_default_plan_profile.h

```diff
@@ -63,20 +63,25 @@ class DescartesDefaultPlanProfile : public DescartesPlanProfile<FloatType>
 
 #ifndef SWIG
   DescartesEdgeEvaluatorAllocatorFn<FloatType> edge_evaluator{ nullptr };
+  DescartesStateEvaluatorAllocatorFn<FloatType> state_evaluator{ nullptr };
 
   // If not provided it adds a joint limit is valid function
   DescartesVertexEvaluatorAllocatorFn<FloatType> vertex_evaluator{ nullptr };
 #endif
 
+  bool allow_collision{ false };
```

Comment:
Doxygen.

Reply:
What do you think about removing this parameter? I believe we can get the same behavior from the inverse of `enable_collisions`